ms/second-review-responses.tex

```diff
@@ -166,23 +196,65 @@
 I am fairly pleased overall with the revised version of this article. I commend the authors for the substantial amount of work conducted. This contribution now convincingly demonstrates how important it is to properly account for population structure when individuals are distributed along a spatial continuum.
 \end{quote}
 
+Thanks. We are also fairly pleased with it, overall.
+\plr{this is probably too much =)}
```

Comment:
edit this?

Reply:
too cheeky!

Reply:
LOL I forgot about this but will remove.